bower.json

```diff
@@ -37,7 +37,8 @@
     "algolia-autocomplete.js": "^0.17.0",
     "algolia-autocomplete-no-conflict": "1.0.0",
     "font-awesome": "fontawesome#^4.7.0",
-    "fetch": "^2.0.2"
+    "fetch": "^2.0.2",
+    "waypoints": "^4.0.1"
```

Comment:
Nit: Should we prefer the `npm i` variation as bower is deprecated (although maintained). I think they are doing work to ensure their registry is frozen, but their recommendation is to use `npm`.